src/ucp/api/ucp.h

```diff
@@ -1015,6 +1028,32 @@ typedef struct ucp_params {
 } ucp_params_t;
 
 
+/**
+ * @ingroup UCP_CONTEXT
+ * @brief Lib attributes.
+ *
+ * The structure defines the attributes which characterize
```

Comment:
Sorry for nitpicking: `The structure defines the attributes which characterize` -> `The structure defines the attributes, which characterize`